tests/Functional/EnglishHumanizationTest.php

```diff
@@ -48,14 +48,14 @@ public function humanizationProvider(): Generator {
 		yield 'Interval with unknown end' => [ '2019/', 'From 2019 to unknown' ];
 		yield 'Interval with unknown start' => [ '/2021', 'From unknown to 2021' ];
 
-		yield 'Year approximate' => [ '2019~', 'Circa 2019' ];
-		yield 'Year uncertain' => [ '2019?', 'Maybe 2019' ];
-		yield 'Year uncertain approximation' => [ '2019%', 'Maybe circa 2019' ];
+		yield 'Year approximate' => [ '2019~', 'Circa 2019 (date is approximate)' ];
+		yield 'Year uncertain' => [ '2019?', 'Maybe 2019 (date is uncertain)' ];
+		yield 'Year uncertain approximation' => [ '2019%', 'Maybe circa 2019 (date is approximate and uncertain)' ];
 
-		yield 'Month approximate' => [ '2019-04~', 'Circa April 2019' ];
-		yield 'Month uncertain' => [ '2019-04?', 'Maybe April 2019' ];
-		yield 'Day approximate' => [ '2019-04-01~', 'Circa April 1st, 2019' ];
-		yield 'Day uncertain' => [ '2019-04-01?', 'Maybe April 1st, 2019' ];
+		yield 'Month approximate' => [ '2019-04~', 'Circa April 2019 (date is approximate)' ];
+		yield 'Month uncertain' => [ '2019-04?', 'Maybe April 2019 (date is uncertain)' ];
+		yield 'Day approximate' => [ '2019-04-01~', 'Circa April 1st, 2019 (date is approximate)' ];
+		yield 'Day uncertain' => [ '2019-04-01?', 'Maybe April 1st, 2019 (date is uncertain)' ];
```

Comment:
@thomas-topway-it can you also add tests for more complex cases like the one you shared in the chat (`maybe circa December 1st, 1985 (year and day are uncertain and month is approximate and uncertain)`)

Reply:
Ok, I was thinking to do that

Reply:
There are no tests for these simple cases yet:
* year uncertain: ?2004-06-11
* month uncertain: 2004-?06-11
* day uncertain: 2004-06-?11